### PR TITLE
a

### PR DESCRIPTION
**Pull request by isvale**

### CODE REVIEW

File: src/trailing-slashes.txt [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
 /nitro-gift
+/nitro-promo
 /nitrodiscord
 /nitrodrop
 /nitrogift
 /nitrogive
-/newyear/freenitro
 /promo
+/promo
+/promotion
```
Comment (iam-py-test) [inline src/trailing-slashes.txt:29]: These two seem way to broad - there are many websites which use these

File: src/trailing-slashes.txt [hunk 2]
```diff
@@ -1,24 +1,32 @@
...
 /airdrop
 /billing
+/claim
+/claim-nitro
 /discordnitro
 /drop
+/free-gift
 /free-nitro
+/freenitro
 /get
```
Comment (iam-py-test) [inline src/trailing-slashes.txt:12]: This is way to broad.

**Comment by DevSpen** [comment]
@isvale Please resolve conflicts and remove the broad trailing slashes `/promo` and `/promotion`. I'll merge when that is done.

**Comment by Rubenennj** [comment]
Damn

File: src/trailing-slashes.txt (hunk 2 above)
Comment (iam-py-test) [reply src/trailing-slashes.txt:12]: I meant /get

Comment (DevSpen) [reply src/trailing-slashes.txt:12]: > I meant /get
I've removed it

File: src/trailing-slashes.txt (hunk 1 above)
Comment (isvale) [reply src/trailing-slashes.txt:29]: what does broad mean?

Comment (iam-py-test) [reply src/trailing-slashes.txt:29]: I mean that a lot of legitimate websites who have promotions or promotional offers will be broken.
Also, the rule `/promo` seems redundant anyway

Comment (DevSpen) [reply src/trailing-slashes.txt:29]: Yeah, those could cause false-positives.

**Comment by DevSpen** [comment]
@isvale Can you please finish this one up? 😋

**Comment by isvale** [comment]
idk know how github works so hopefully i fixed the conflicts 🤷‍♂️

**Comment by iam-py-test** [comment]
> idk know how github works so hopefully i fixed the conflicts 🤷‍♂️

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests
@DevSpen can you fix the conflicts?

**Comment by DevSpen** [comment]
> > idk know how github works so hopefully i fixed the conflicts 🤷‍♂️
> 
> https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests @DevSpen can you fix the conflicts?

Sure, I'll fix this one up

**Pull request merged**